frontend/public/components/utils/breadcrumbs.ts

```diff
@@ -13,13 +13,26 @@ export const getBreadcrumbPath = (match: any, customPlural?: string) => {
   return `/k8s/cluster/${customPlural || match.params.plural}`;
 };
 
-export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => [
-  {
-    name: `${kindObj.labelPlural}`,
-    path: getBreadcrumbPath(match),
-  },
-  {
-    name: i18next.t('details-page~{{kind}} details', { kind: kindObj.label }),
-    path: `${match.url}`,
-  },
-];
+export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => {
+  return match.url.startsWith('/k8s/cluster/config')
+    ? [
+        {
+          name: 'Global Configuration',
+          path: '/settings/cluster/globalconfig',
+        },
+        {
+          name: i18next.t('details-page~{{kind}} details', { kind: kindObj.label }),
+          path: `${match.url}`,
+        },
+      ]
+    : [
+        {
+          name: `${kindObj.labelPlural}`,
+          path: getBreadcrumbPath(match),
+        },
+        {
+          name: i18next.t('details-page~{{kind}} details', { kind: kindObj.label }),
+          path: `${match.url}`,
+        },
```

Comment:
I'd avoid repeating this twice since it's identical to above.

Reply:
fixed